kafka-client-examples/e2e-test/docker/antithesis/async/config/volumes/app-rs3.properties

```diff
@@ -19,3 +19,4 @@ responsive.rs3.hostname=rs3
 responsive.rs3.port=50051
 responsive.rs3.logical.store.mapping=e2e:b1a45157-e2f0-4698-be0e-5bf3a9b8e9d1
 responsive.rs3.tls.enabled=false
+responsive.rs3.retry.timeout.ms=1800000
```

Comment:
what default are we using? should probably be set to like half the max poll interval

Reply:
I had it at 30s, which might be on the low side. Do we use the default poll interval of 5min?

Reply:
I changed the default to 2 minutes. That is close to half the poll interval and matches the default producer delivery timeout.